<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>About</class>
    <widget class="QDialog" name="aboutWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>320</width>
                <height>240</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>About...</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../qfm1000.qrc">
                <normaloff>:/icons/application</normaloff>:/icons/application
            </iconset>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="text">
                        <string>qFM1000</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="imageLabel">
                    <property name="text">
                        <string/>
                    </property>
                    <property name="pixmap">
                        <pixmap>:/icons/about.svg</pixmap>
                    </property>
                    <property name="scaledContents">
                        <bool>false</bool>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="textLabel">
                    <property name="text">
                        <string>textLabel</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Close</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../qfm1000.qrc"/>
    </resources>
    <connections/>
</ui>
